docs/tutorials/create-a-service-to-rotate-an-image.md

```diff
@@ -505,7 +505,8 @@ uvicorn main:app --reload --host localhost --port 9090 # (1)!
    from the one used by the Core engine.
 
 !!! Note
-    The [Core engine](../reference/core-engine.md#start-the-service-locally-with-node)
+    The
+    [Core engine](../reference/core-engine.md#start-the-service-locally-with-node)
```

Comment:
Is this line break useful ?

Reply:
It is the work of [mdwrap](https://docs.swiss-ai-center.ch/reference/mdwrap/).

As the URL is longer than 80 characters, it has moved it on its own line. It's better to leave it this way.